### DISCUSSION TITLE
OpenSSL library errors printed to stdout

### DISCUSSION THREAD

**Discussion by dev31337** [openssl/openssl#22389] [General Discussion]
Hi,

OpenSSL library is printing errors on stdout, how can I ask OpenSSL library to not print any error or to call a custom error function ?

Regards.

**Comment by nhorman** [comment]
Openssl records errors to an internal buffer that applications can flush to various locations using the ERR API (see ERR_print_error_fp as an example).

If you have an application printing error messages from the openssl libs, then the application is calling one of those functions to do so.

Is this the openssl app printing messages,.or something you're writing/maintaining?

**Comment by dev31337** [comment]
Thank you for your help, you were right the application was calling ERR_print_error_fp(stderr).
